packages/mobile/src/components/track-details-tile/TrackDetailsTile.tsx

```diff
@@ -0,0 +1,146 @@
+import type { ID } from '@audius/common'
+import {
+  SquareSizes,
+  getDogEarType,
+  isPremiumContentCollectibleGated,
+  usePremiumContentAccess,
+  cacheUsersSelectors,
+  cacheTracksSelectors
+} from '@audius/common'
+import { View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import IconCollectible from 'app/assets/images/iconCollectible.svg'
+import IconSpecialAccess from 'app/assets/images/iconSpecialAccess.svg'
+import { DogEar, Text } from 'app/components/core'
+import { TrackImage } from 'app/components/image/TrackImage'
+import UserBadges from 'app/components/user-badges'
+import { makeStyles, flexRowCentered, typography } from 'app/styles'
+import { spacing } from 'app/styles/spacing'
+import { useColor } from 'app/utils/theme'
+
+const { getUser } = cacheUsersSelectors
+const { getTrack } = cacheTracksSelectors
+
+const messages = {
+  collectibleGated: 'COLLECTIBLE GATED',
+  specialAccess: 'SPECIAL ACCESS'
+}
+
+const useStyles = makeStyles(({ spacing, palette }) => ({
+  root: {
+    backgroundColor: palette.neutralLight10,
+    borderWidth: 1,
+    borderColor: palette.neutralLight8,
+    borderRadius: spacing(2),
+    width: '100%'
+  },
+  trackDetails: {
+    ...flexRowCentered(),
+    padding: spacing(4),
+    gap: spacing(4)
+  },
+  trackImage: {
+    width: spacing(22),
+    height: spacing(22),
+    borderRadius: spacing(1),
+    borderColor: palette.neutralLight8
```

Comment:
Just a thought: On web, I added the border and text semantic color names, so this kind of thing _could_ be `palette.borderDefault` if you wanted to add those values at some point as well. Not necessary for this PR though.

Reply:
yeah agree! not gonna worry about that for now tho.